frame/staking/src/mock.rs

```diff
@@ -88,7 +75,12 @@ impl sp_runtime::BoundToRuntimeAppPublic for OtherSessionHandler {
 
 pub fn is_disabled(controller: AccountId) -> bool {
 	let stash = Staking::ledger(&controller).unwrap().stash;
-	SESSION.with(|d| d.borrow().1.contains(&stash))
+	let validator_index = match Session::validators().iter().position(|v| *v == stash) {
+		Some(index) => index as u32,
+		None => return false,
+	};
+
+	Session::disabled_validators().contains(&validator_index)
```

Comment:
Is this the only place we read `DisabledValidators`? I think I am missing the purpose of this storage item since afaict we keep it updated, but we only use it in this testing helper function.